server/data/data-api/src/main/java/org/apache/james/rrt/lib/Mapping.java

```diff
@@ -186,7 +197,8 @@ public static boolean hasPrefix(String mapping) {
                 || mapping.startsWith(Domain.asPrefix())
                 || mapping.startsWith(Error.asPrefix())
                 || mapping.startsWith(Forward.asPrefix())
-                || mapping.startsWith(Group.asPrefix());
```

Comment:
Here also we could propose a generic version to avoid requiring to modify this method  upon *each new Mapping type*, similary to **detectType**